src/app/containers/SheriffList.tsx

```diff
@@ -38,7 +44,9 @@ class SheriffList extends React.Component<CompositeProps> {
       sheriffs = [],
       loading = true,
       SheriffRenderer,
-      showSheriffProfileModal
+      showSheriffProfileModal,
+      // tslint:disable-next-line:no-shadowed-variable
+      sheriffLoanMap = {}
     } = this.props;
 
     if (loading) {
```

Comment:
We should make a loading component.  Even if for right now it just is `<div>Loading...</div>`